docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -189,14 +189,12 @@ These steps will need to be repeated if Teleport's user certificate authority is
 </Admonition>
 
 
-1. Get the Teleport user CA certificate by running:
+Get the Teleport user CA certificate by running the following in the Windows machine where you can manage your group policy.
 
 ```code
-$ tctl auth export --type=windows > user-ca.cer
+$ curl https://teleport.example.com/webapi/auth/export?type=windows > user-ca.cer
```

Comment:
Desktop access supports Windows Server 2012R2 and newer.

Reply:
It seems only Windows Server 2019 onwards has `curl` binary by default
 
So, should we update the message to include both methods?
Or suggest to download and then transfer a file from a Linux host?
Depending on how we phrase it, it might get confusing 🤔

Something like 
```
Get the Teleport user CA certificate by running the following in the Windows machine where you can manage your group policy.
```code
$ curl 'https://teleport.example.com/webapi/auth/export?type=windows' > user-ca.cer
```

If `curl` is not available, use another host to get the file and then copy it into the Windows machine.
```

Reply:
@ptgott Can you please give your input to the issue above?
Do you think the message is clear?

Reply:
Technically, the previous approach required tctl, which also isn't available on Windows, so this is really no worse.

Reply:
I'll keep it as is then :+1: